docker/config/application.yaml

```diff
@@ -45,7 +45,7 @@ fhirdata:
   rowGroupSizeForParquetFiles: 33554432   # 32mb
   viewDefinitionsDir: "config/views"
   createParquetViews: true
-  sinkDbConfigPath: "config/hapi-postgres-config_local_views.json"
+  #sinkDbConfigPath: "config/hapi-postgres-config_local_views.json"
```

Comment:
I think there are some E2E tests that rely on this. If we want to turn this off by default (which is okay with me) we should still override it in the relevant E2E test to turn this on. Another option is to leave this enabled by default (which is also fine with me).

Reply:
Reverted.